plugins/reactTo.js

```diff
@@ -1,28 +1,36 @@
-var MQ = require('../index');
-var screenSize = require('../_common').screenSize;
+var MQ = require('../index')
+var screenSize = require('../_common').screenSize
 
-var callbackList = [];
+var callbackList = []
 
 MQ.prototype.reactTo = function (query, callback) {
 	if (typeof query !== 'function') {
-		throw new Error('First argument should be a function that returns the result of an mq-js screen check, eg: ()=> mq.min(1000)')
+		throw new Error(
+			'First argument should be a function that returns the result of an mq-js screen check, eg: ()=> mq.min(1000)'
+		)
 	}
 	if (typeof callback !== 'function') {
-		throw new Error('Second argument should be a function that you wish to run when a change in screen size is detected.')
+		throw new Error(
+			'Second argument should be a function that you wish to run when a change in screen size is detected.'
+		)
 	}
 
-	var oldStatus = query();
-	callbackList.push(callback);
+	var oldStatus = query()
+	callbackList.push(callback)
 
-	window.addEventListener("resize", check_query(query, oldStatus), true);
+	check_query(query, null)()
```

Comment:
FYI, adding this line was the only actual change I made to the plugin in this PR.

The rest is all either prettier auto-formatting or upgrading the immensely outdated build process.